### ISSUE TITLE
batch_size argument validation to avoid NaN.

### ISSUE THREAD

**Issue by wenxueru** [volcengine/verl#405]
In `verl/workers/critic/dp_critic.py`:
```python
micro_batches = mini_batch.split(self.config.ppo_micro_batch_size_per_gpu)
self.gradient_accumulation = self.config.ppo_mini_batch_size // self.config.ppo_micro_batch_size_per_gpu
```
If `self.config.ppo_mini_batch_size < self.config.ppo_micro_batch_size_per_gpu`, self.gradient_accumulation will be 0.
Then the loss computation will result in NaN.
```python
loss = vf_loss / self.gradient_accumulation
```
There should be validation of `ppo_mini_batch_size` and `ppo_micro_batch_size_per_gpu`.

**Comment by vermouth1992** [comment]
Yes, please see #382
